javascript/ql/src/Security/CWE-020/HostnameRegexpShared.qll

```diff
@@ -0,0 +1,105 @@
+import javascript
```

Comment:
This deserves a minor module docstring. 

(can any of the predicates in this file be made private?)